app/models/feedbook/employee_has_session.rb

```diff
@@ -0,0 +1,7 @@
+module Feedbook
+  class EmployeeHasSession < ApplicationRecord
+    belongs_to :session
+    belongs_to :manager, class_name: 'Employee'
+    belongs_to :employee, class_name: 'Employee'
```

Comment:
I'm pretty sure this is not needed, I think rails will infer that the class_name for this relation is `Employee` based on the symbol